cmd/kubectl-direct_csi/drives_accesstier_set.go

```diff
@@ -96,10 +96,9 @@ func setAccessTier(ctx context.Context, accessTier directcsi.AccessTier) error {
 	ctx, cancelFunc := context.WithCancel(ctx)
 	defer cancelFunc()
 
-	directCSIClient := client.GetDirectCSIClient()
 	return processFilteredDrives(
 		ctx,
-		directCSIClient.DirectCSIDrives(),
+		client.GetUnversionedDirectCSIDriveClientset(),
```

Comment:
We don't need to pass this client argument anymore. The variable is initialized in the client package. We can just use it locally in that package. 

We can get rid of the client argument entirely for both processFiltered* and getFiltered*List